pages/index.tsx

```diff
@@ -23,21 +28,120 @@ const Home: NextPage = () => {
+        }}
+      >
+        <Image
+          src={candidates}
+          alt="Background Image"
+          layout="fill"
+          objectFit="cover"
+          quality={100}
+        />
+        <div
+          style={{
+            position: 'relative',
+            backgroundColor: 'rgba(244, 244, 244, 0.7)',
+            margin: 'auto',
+          }}
+        >
+          <p className="bg-purple-200 pl-10 text-left font-bold text-gray-700">
+            Find the best job
+          </p>
+          <form
+            style={{
+              position: 'relative',
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'space-between',
+              gap: '1rem',
+              padding: '1rem',
+            }}
+            onSubmit={handleSearch} 
+          >
+            <div style={{ flex: '1' }}>
+              <input
+                type="text"
+                placeholder="Job Title"
+                style={{
+                  width: '100%',
+                  padding: '0.5rem',
+                  border: '1px solid #E5E7EB',
+                  borderRadius: '0.375rem',
+                  outline: 'none',
+                }}
+                value={titleInput}
+                onChange={handleTitleInputChange}
+              />
+            </div>
+            <div style={{ flex: '1' }}>
+              <input
+                type="text"
+                placeholder="Location"
+                style={{
+                  width: '100%',
+                  padding: '0.5rem',
+                  border: '1px solid #E5E7EB',
+                  borderRadius: '0.375rem',
+                  outline: 'none',
+                }}
+                value={locationInput}
+                onChange={handleLocationInputChange}
+              />
+            </div>
+            <button
+              type="submit"
+              style={{
+                backgroundColor: '#3B82F6',
+                color: 'white',
+                padding: '0.5rem 1rem',
+                borderRadius: '0.375rem',
+                cursor: 'pointer',
+              }}
+            >
+              Search
+            </button>
+          </form>
+        </div>
+      </div>
+      <h1 className="mx-auto mb-6 flex max-w-[80%] justify-center text-center text-2xl font-bold md:mb-8 md:max-w-[300px] md:max-w-[600px] md:pt-24 md:text-4xl">
         The #1 platform for building transparent futures
       </h1>
-      <div className=' mb-12 flex flex-row items-center justify-center sm:mx-auto sm:px-6 md:mb-16'>
+      <div className=" mb-12 flex flex-row items-center justify-center sm:mx-auto sm:px-6 md:mb-16">
         <Stats stats={stats} />
       </div>
 
-      <div className='container'>
-        <div className='text-2xl font-semibold'>
-          Showing {jobArr.length} Jobs
+      <div className="container">
+        <div className="text-2xl font-semibold">
+          Showing {filteredJobs.length || jobArr.length} Job(s)
```

Comment:
if a search returns no results then we probs want to reflect that in the UI i.e no results found

Reply:
surely this is done. would resend